profile-service/src/main/java/com/transformuk/hee/tis/profile/web/rest/RoleResource.java

```diff
@@ -39,6 +40,8 @@
 public class RoleResource {
 
   private static final String ENTITY_NAME = "role";
+  protected static final Set<String> restrictedRoles = Set.of("RVOfficer",
+      "Machine User", "HEE");
```

Comment:
This works for the current story.

Thinking of how we'd refactor clients that currently use the concept of restricted roles, which I think is just User Management, would this be better placed in part of the API module or would we expose what the restricted roles are via. an endpoint that clients could use?

Reply:
As discussed, we will remove the restrictedroles static collection from UserManagement.